components/dropdownlist/filter.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Filter
+page_title: DropDownList - Filter
+description: Filtering in the DropDownList for Blazor.
+slug: components/dropdownlist/filter
+tags: telerik,blazor,drop,down,list,dropdownlist,filter
+published: True
+position: 5
+---
+
+# DropDownList Filter
+
+The Filter in the DropDownList component is an input shown at the top of the dropdown popup, used for filtering the underlying datasource.
+
+To enable filtering, set the `Filterable` parameter to `true`.
+
+Filtering ignores casing and the default filter operator is `starts with`. Filtering looks in the `TextField`, and the filter is reset when the dropdown closes. You can choose a different operator through the `FilterOperator` parameter that takes a member of the `Telerik.Blazor.StringFilterOperator` enum.
```

Comment:
Consider putting "You can choose a different operator" on a separate line (paragraph)

Reply:
Done.